modules/otel-troubleshoot-metrics.adoc

```diff
@@ -42,3 +40,19 @@ $ oc port-forward <collector_pod>
 ----
 
 . Access the metrics endpoint at `+http://localhost:8888/metrics+`.
+
+Alternatively, you can enable the creation of PodMonitors or ServiceMonitors (depending on the deployment mode of the OpenTelemetryCollector) to scrap the internal metrics by setting to `true` the `enableMetrics` setting.
```

Comment:
@iblancasa, which option (line 42 or line 44+) do you recommend to users? Do you have a clear preference?

Reply:
One or another will be enabled automatically depending on how the OTEL Collector was deployed (if it is `Sidecar` mode, for instance, a `PodMonitor` will be created but for `Deployment` mode a `ServiceMonitor` will be created.)

Reply:
:sweat_smile: Sorry for my lack of clarity: I mean whether to access the metrics endpoint or scrap the internal metrics by setting to `true` the `enableMetrics` setting.

Reply:
Oh! Sorry.

I would suggest enabling the `enableMetrics` field because, with that, users will not need to change anything on their side.